src/components/AccountPickerItem.jsx

```diff
@@ -35,12 +36,18 @@ export const AccountPickerItem = ({
         <span>{accountName}</span>
         {nameAndLoginDiffer && <small>{accountLogin}</small>}
       </div>
-      {hasError && (
-        <div className={styles['col-account-picker-button-error']}>
-          <span>
-            {getErrorTitle(t, error, key => `connection.error.${key}.title`)}
-          </span>
-          <Icon icon="warning" />
+      {hasError &&
+        !hasUpdate && (
```

Comment:
Isn't it weird that we hide the error if there is an update ?

Reply:
Yep, but since this is the account item here, the update is more important to be displayed (since it could resolve an error) but the error will still displayed in the account modal.